tests/conftest.py

```diff
@@ -0,0 +1,114 @@
+# pylint: disable=redefined-outer-name,too-many-statements
+"""Initialise a text database and profile for pytest."""
+
+import os
+from pathlib import Path
+
+import pytest
+
+from aiida.common import exceptions
+from aiida.common.folders import SandboxFolder
+from aiida.engine.utils import instantiate_process
+from aiida.manage.manager import get_manager
+from aiida.orm import InstalledCode, load_code
+from aiida.plugins import CalculationFactory
+
+pytest_plugins = ["aiida.manage.tests.pytest_fixtures"]  # pylint: disable=invalid-name
+
+
+@pytest.fixture(scope="function", autouse=True)
+def clear_database_auto(aiida_profile_clean):  # pylint: disable=unused-argument
+    """Automatically clear database in between tests."""
+
+
+@pytest.fixture(scope="session")
+def filepath_tests():
+    """Return the absolute filepath of the `tests` folder.
+
+    .. warning:: if this file moves with respect to the `tests` folder, the
+    implementation should change.
+
+    :return: absolute filepath of `tests` folder which is the basepath for all test
+    resources.
+    """
+    return Path(__file__).resolve()
+
+
+@pytest.fixture
+def filepath_fixtures(filepath_tests):
+    """Return the absolute filepath to the directory containing the file `fixtures`."""
+    return Path(filepath_tests / "fixtures")
+
+
+@pytest.fixture(scope="function")
+def fixture_sandbox():
+    """Return a `SandboxFolder`."""
+
+    with SandboxFolder() as folder:
+        yield folder
+
+
+@pytest.fixture
+def fixture_localhost(aiida_localhost):
+    """Return a localhost `Computer`."""
+    localhost = aiida_localhost
+    localhost.set_default_mpiprocs_per_machine(1)
+    return localhost
+
+
+@pytest.fixture(scope="function")
+def janus_code(aiida_local_code_factory):
+    """Get the janus code."""
+    janus_path = os.environ.get("JANUS_PATH")
```

Comment:
this does not get the path as expected... 
if you set janus_path = "janus"

alll will work but in theory env variable shall work correctly